kata06/app/views/employees/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Email</th>
+      <th>Posicion</th>
+      <th>Num empleado</th>
+      <th>Private num</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @employees.each do |e| %>
```

Comment:
you can be more descriptive with the name e

Reply:
ok! i will put employee!